src/sentry/utils/sdk.py

```diff
@@ -23,6 +24,11 @@
     "outcomes_consumer.py",
 )
 
+# URLs that should always be sampled
+SAMPLED_URL_NAMES = [
+    "sentry-api-0-organization-releases",
+]
```

Comment:
One thing you might think about (even though it's overkill for now) is making this a dictionary of url name to sample rate. That way if we ever want to add endpoints which don't get sampled at 100%, we can do that easily. 

(Even if you don't do that, I'd still make it a set rather than a list, to get the constant lookup time. Not an issue for a list of one, of course, and not even really much of an issue for a list of 10 or 20 or 50, but why not get the time savings?)

Reply:
For now I don't think there's any need to change this to a dictionary, and is a pretty easy change if there's eventually a use case for that 👍. But good idea on making this a set instead.